### PR TITLE
Docs: Add CodeQL standard packs and bundle to CodeQL tools page

### PR DESCRIPTION
**Pull request by adityasharad**
Include links to changelogs, releases, and source code.
See internal issue for context.

### CODE REVIEW

**Comment by mchammer01** [comment]
Covering for the docs first responder here 👋🏻 - I added this PR to our board for review by the Docs team. Thanks for your patience 🙇🏻‍♀️ 😅

File: docs/codeql/codeql-overview/codeql-tools.rst [hunk 1]
```diff
@@ -13,9 +13,40 @@ CodeQL command-line interface
...
+- ``codeql/ruby-queries`` (`changelog <https://github.com/github/codeql/tree/codeql-cli/latest/ruby/ql/src/CHANGELOG.md>`__, `source <https://github.com/github/codeql/tree/codeql-cli/latest/ruby/ql/src>`__)
+- ``codeql/ruby-all`` (`changelog <https://github.com/github/codeql/tree/codeql-cli/latest/ruby/ql/lib/CHANGELOG.md>`__, `source <https://github.com/github/codeql/tree/codeql-cli/latest/ruby/ql/lib>`__)
+
+For more information, see ":ref:`About CodeQL packs <about-codeql-packs>`."
+
+CodeQL bundle
+-----------------------------
+
+The CodeQL bundle consists of the CodeQL CLI together with the standard CodeQL query and library packs
+maintained by GitHub. It can be downloaded from `GitHub releases <https://github.com/github/codeql-action/releases>`__.
```
Comment (mchammer01) [inline docs/codeql/codeql-overview/codeql-tools.rst:48]: Same comment as above, perhaps, although not as critical:
```suggestion
maintained by GitHub. The bundle can be downloaded from `GitHub releases <https://github.com/github/codeql-action/releases>`__.
```

**Review by mchammer01 — APPROVED** [review]
@adityasharad - this LGTM ✨ 
Thanks for your patience!
I initially didn't see the difference between the source queries for a language and the `all` variant`, but I've since realized the directories aren't the same 😅 
A couple of minor suggestions to improve readability (feel free to ignore if you don't agree)

**Comment by adityasharad** [comment]
Thanks! Suggestions incorporated -- this will need a fresh approval.

**Review by mchammer01 — APPROVED** [review]
Thanks for addressing my comments 💖 
This is ready to go :shipit:

**Pull request merged**